config/metadata/configMeta.yaml

```
@@ -1258,6 +1258,7 @@ groups:
           `MB` and `GiB`) and Kubernetes units (such as `M` and `Gi`) are
           supported. Fractional values with a suffix are supported. If
           `AvailableMemory` is set, `Collections.MaxAlloc` must not be defined.
+          A useful value for this setting is about 85% of the pod's total memory.
```

I think 85% is a bit conservative. We usually recommend 90-95% so it leaves some headroom to the OS but still gets most of it. I also like Kent's phrasing for leaving some headroom for the OS.

```suggestion
          A good starting value for this setting is 90% of the pod's total memory.
```

---

updated based on another thread, and did 85-90% with a specific note of 1-2GB.